src/Microsoft.AspNetCore.Mvc.Localization/project.json

```diff
@@ -1,5 +1,10 @@
 {
-  "description": "Features that enable globalization & localization of MVC applications.",
+  "description": "ASP.NET Core MVC features that enable globalization and localization of applications.\r\nCommonly used types:\r\nMicrosoft.AspNetCore.Mvc.Localization.IHtmlLocalizer\r\nMicrosoft.AspNetCore.Mvc.Localization.IHtmlLocalizer<TResource>",
```

Comment:
I definitely meant what I put there, but I could be wrong. But I'm thinking I should at least add `Microsoft.AspNetCore.Mvc.Localization.IViewLocalizer`? And keep the other ones there?